web/client/components/TOC/fragments/settings/FeatureInfo.jsx

```diff
@@ -84,12 +84,15 @@ export default class extends React.Component {
      * @return {object} info formats
      */
     supportedInfoFormats = () => {
+        const excludedFormatsWfs = ['TEXT', 'HTML'];
         const availableInfoFormats =  this.props.element?.infoFormats || [];
+        const supportedWfsFormats = Object.fromEntries(Object.entries(this.props.defaultInfoFormat).filter(([key]) => !excludedFormatsWfs.includes(key)));
+        const formats = this.props.element.type === 'wfs' ? supportedWfsFormats : this.props.defaultInfoFormat;
```

Comment:
can you add a unit tests that will cover the list of info formats used by "wfs" and maybe another one like "wms" ?

you can render this comp and check the items rendered in the list